tests/MercadoPago/Resources/Mocks/Response/Order/get_order_response.json

```
@@ -0,0 +1,36 @@
+{
+  "id": "01JD2P9GGXAPBDGG6YT90N77M3",
+  "type": "online",
+  "processing_mode": "automatic",
+  "external_reference": "ext_ref_1234",
+  "total_amount": "200.00",
+  "site_id": "MLB",
+  "status": "processed",
+  "status_detail": "accredited",
+  "created_date": "2024-11-19T17:07:31.486Z",
+  "last_updated_date": "2024-11-19T17:07:33.068Z",
+  "payer": {
+      "email": "test_1731354550@testuser.com"
+  },
+  "transactions": {
+      "payments": [
+          {
+              "id": "pay_01JD2P9GGXAPBDGG6YT92J5305",
+              "amount": "200.00",
+              "reference_id": "0001ozsb5o",
+              "status": "processed",
+              "status_detail": "accredited",
+              "payment_method": {
+                  "id": "master",
+                  "type": "credit_card",
+                  "token": "6fa1cc12bb30405599c944b8adecbaee",
```

Aqui podemos deixar sem o token real, por precaução de segurança.
```suggestion
                  "token": "card_token",
```